amd_openvx/openvx/ago/ago_kernel_api.cpp

```diff
@@ -1868,8 +1868,11 @@ int ovxKernel_HarrisCorners(AgoNode * node, AgoKernelCommand cmd)
     //       use VX_KERNEL_AMD_HARRIS_SCORE_* kernels to compute Vc
     //       use VX_KERNEL_AMD_HARRIS_MERGE_SORT_AND_PICK_XY_HVC kernel for final step
     //       disable buffer merging for HarrisCorners
-    if(node->attr_affinity.device_type == AGO_KERNEL_FLAG_DEVICE_GPU){
-        agoSetEnvironmentVariable("AGO_BUFFER_MERGE_FLAGS", "1");
+    char textBuffer[1024];
```

Comment:
Can we enclose this workaround with `#if ENABLE_OPENCL` flag for both set and unset?

Reply:
@kiritigowda added the guards for the set routine. The unset routine is inside the [agoGpuAllocBuffers](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/blob/master/amd_openvx/openvx/ago/ago_drama_alloc.cpp#L63) function which already has the GPU guards.